content/en/getting_started/support/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Getting started with Datadog Support
+kind: documentation
+further_reading:
+- link: "https://docs.datadoghq.com/agent/troubleshooting/"
+  tag: "Documentation"
+  text: "Agent Troubleshooting"
+---
+
+## Overview
+
+Datadog provides two primary channels for customers seeking support:
+   - [Opening a support ticket][1].
+   - Chatting live with Datadog Technical Support Engineers through a text chat.
+
+While both options focus on providing quick and effective solutions, one platform may be better suited depending on the timing, nature, and urgency of the request.
+
+This guide provides best practices for reaching out to the Support team, and guidelines for determining the support channel that’s right for you.
+
+## Prerequisites
+
+For the most efficient resolution of an issue, be ready to provide any of the relevant information and resources listed below:
+
+   - Organization name (if you have access to [more than one][2])
+   - [Datadog Site][3]
+   - Links to any pages demonstrating the issue in your Datadog account
+   - Screenshots or screen recordings of the behavior in question
+   - Steps that can be taken to replicate the issue
+   - If the issue is related to the functioning of the Datadog Agent, an [Agent flare][4]
+   - Links to any documentation pages used
+
+## Chat or support ticket?
+
+Use the table below to determine whether to open a [support ticket][1] or reach out on chat. Chat is available any business day between the hours of 10:00 and 19:00 ET.
```

Comment:
I'm not sure why, but this link doesn't work for me in the staging preview (even though the link itself at the bottom of the file is correct).